docs/Understanding-FLINT/data-preparation/index.rst

```diff
@@ -102,10 +106,12 @@ without compromising the processing capacity.
 
 Vector
 ~~~~~~
-
-Vectors are also a format of spatial files for GIS, however, unlike
-rasters, the information relates to points, lines or polygons, rather
-than pixels. For the FLINT, as a minimum, vector files must…..[TBC]
+Vectors are another format of spatial files for GIS. They are composed of 
+points, lines and polygons which are generated by a set of commands or 
+mathematical statements. The Shapefile format is a popular geospatial vector data format. 
+To represent discrete features like a river, road, building etc, 
+vectors are preferred.
+For the FLINT, as a minimum, vector files must…..[TBC]
```

Comment:
Is this piece still incomplete?

Reply:
Yes, I think it is. 
Since it was present before, I did not make any change to it

Reply:
Suggest removing incomplete sections.

Reply:
Done